README.md

```diff
@@ -1 +1,64 @@
-# microverse
\ No newline at end of file
+# Hello Microverse
+
```

Comment:
- [ ] In your README file kindly add the `table of content` to it, and all the necessary headers with their respective content describing what they entail. headers include:

  - `Key Feature`
  - `Future Features`

Also kindly remove the live server link as you don't have any deployed link for your project now. You can simply replace the section with a text stating `link not available`.

Reply:
Thank you for your suggestion, I will make the required changes.